x-pack/test/functional/apps/aiops/change_point_detection.ts

```diff
@@ -0,0 +1,92 @@
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.testResources.deleteIndexPatternByTitle('ft_ecommerce');
+    });
+
+    it(`loads the change point detection page`, async () => {
+      // Start navigation from the base of the ML app.
+      await ml.navigation.navigateToMl();
+      await elasticChart.setNewChartUiDebugFlag(true);
+      await aiops.changePointDetectionPage.navigateToIndexPatternSelection();
+      await ml.jobSourceSelection.selectSourceForChangePointDetection('ft_ecommerce');
+      await aiops.changePointDetectionPage.assertChangePointDetectionPageExists();
+    });
+
+    it('detects a change point when no split field is selected', async () => {
+      await aiops.changePointDetectionPage.clickUseFullDataButton();
+      await aiops.changePointDetectionPage.selectMetricField(0, 'products.discount_amount');
+      const result = await aiops.changePointDetectionPage.getTable(0).parseTable();
+      expect(result.length).to.eql(1);
+      expect(parseInt(result[0].pValue, 10)).to.eql(0);
+      expect(result[0].type).to.eql('distribution_change');
+
+      await elasticChart.waitForRenderComplete('aiopChangePointPreviewChart > xyVisChart');
+      const chartState = await elasticChart.getChartDebugData(
+        'aiopChangePointPreviewChart > xyVisChart',
+        0,
+        5000
+      );
+      if (!chartState) {
+        throw new Error('Preview chart debug state is not available');
+      }
+      expect(chartState.annotations![0].data.details).to.eql('distribution_change');
+      expect(chartState.annotations![0].domainType).to.eql('xDomain');
+      expect(chartState.lines![0].points.length).to.be.above(30);
+    });
+
+    it('shows multiple results when split field is selected', async () => {
+      await aiops.changePointDetectionPage.clickUseFullDataButton();
+      await aiops.changePointDetectionPage.selectMetricField(0, 'products.discount_amount');
+      await aiops.changePointDetectionPage.selectSplitField(0, 'geoip.city_name');
+      const result = await aiops.changePointDetectionPage.getTable(0).parseTable();
+      expect(result.length).to.eql(5);
+      // assert asc sorting by p_value is applied
+      expect(parseFloat(result[0].pValue)).to.be.lessThan(parseFloat(result[4].pValue));
+    });
+
+    it('supports a quick filter actions', async () => {
+      await aiops.changePointDetectionPage
+        .getTable(0)
+        .invokeAction(0, 'aiopsChangePointFilterForValue');
+      const resultFor = await aiops.changePointDetectionPage.getTable(0).parseTable();
+      expect(resultFor.length).to.eql(1);
+    });
+
+    it('allows change point selection for detailed view', async () => {
+      await aiops.changePointDetectionPage.getTable(0).selectAllRows();
+      await aiops.changePointDetectionPage.viewSelected();
```

Comment:
Is there any assertion we can do here to check that the flyout displays all selected series?

Reply:
added in https://github.com/elastic/kibana/pull/158164/commits/96a798305a1abfa5f43d2308627aefa5d897d5fb